site/en/docs/extensions/mv3/overview/index.md

```diff
@@ -110,19 +164,17 @@ or by pressing `Ctrl+Shift+F` on your keyboard.
 ## What next? {: #How-do-I-start }
 
 1.  Follow the [Getting Started tutorial][10]
-2.  Read the [Overview][11]
-3.  Keep up to date by reading the [Chromium blog][12]
-4.  Subscribe to the [chromium-extensions group][13]
-
-## Featured videos {: #Featured-videos }
+1.  Explore the [extension samples][2]
+1.  Subscribe to the [chromium-extensions group][13]
```

Comment:
```suggestion
2.  Subscribe to the [chromium-extensions group][13]
```

Reply:
It's better Markdown practice to make numbered lists go 1., 1., 1., ... and let the processor handle the numbering. Check the staging preview.

Reply:
Nifty!  I saw that the rendering was correct, but didn't realize this was a feature of markdown.  Good to know.